server/helpers/response-messages.js

```diff
@@ -1,16 +1,16 @@
-const userExistsMessage = {
+export const userExistsMessage = {
   status: 'error',
   error: 'The account already exists. Proceed with sign in instead'
 };
 
-const signupInstead = {
+export const signupInstead = {
   status: 'error',
   error: 'User does not exist. Signup instead'
 };
 
-const incorrectCredentials = {
+export const incorrectCredentials = {
   status: 'error',
   error: 'Incorrect email or password'
 };
```

Comment:
these functions are not messages. considering the file name I think they shouldn't be in this file